source/sass/buyers-guide/views/homepage.scss

```diff
@@ -149,7 +149,7 @@
   }
 
   .creepiness-slider {
-    z-index: 2;
+    z-index: 3;
```

Comment:
Filed a [followup ticket](https://github.com/mozilla/foundation.mozilla.org/issues/2560) for this since there are multiple existing rules that need fixing as well.